pages/index.js

```diff
@@ -582,7 +582,6 @@ export default class IndexPage extends React.Component {
     const response = await Actions.health();
     console.log("HEALTH_CHECK", response);
   }
```

Comment:
Why did you delete this line :P

Reply:
Yep my bad I removed it from other pages and index by mistake